docs/pages/packages/jest-environment.mdx

```diff
@@ -53,3 +53,61 @@ it('should return the correct value', () => {
   expect(val).toBe(4)
 })
 ```
+
+## Extend expect
+
+The **@edge-runtime/jest-environment** package comes with a sub-module that contains useful [Jest matchers](https://jestjs.io/docs/expect#expectextendmatchers) to test `Request` and `Response`.
```

Comment:
Maybe it should be `@edge-runtime/jest-expect` or -matchers? This way people can use it outside the jest env

Reply:
Sounds good to me!